impedance/plotting.py

```diff
@@ -42,8 +61,8 @@ def plot_nyquist(ax, freq, Z, fit=False):
     ax.grid(b=True, which='major', axis='both', alpha=.5)
 
     # Change axis units to 10^-3 and resize the offset text
```

Comment:
Comment should probably be updated to reflect that scaling can be arbitrarily set instead of fixed at 10^-3?

Reply:
Good point. Made those changes in the next commit